src/api/common/schema-validator.js

```diff
@@ -11,7 +11,7 @@ const ValidationError = require('./errors').ValidationError;
 
 let SCHEMAS = {};
 
-function isValidAddress(address) {
+function isValidAddress(address: string): boolean {
   return core.UInt160.is_valid(address);
```

Comment:
Do we want values like `skdkfjekcsdkdfkgxcmvnlmmoesdfgvvcdgfkjfv` or `6e3efa86a5eb0a3c5dc9beb3a204783bb00e1913` to be considered as valid addresses here?
Because UInt160 considers it valid.